### PR TITLE
chore(doc): minor corrections in CONTRIBUTING

### PR DESCRIPTION
**Pull request by lemeurherve**
Added goreleaser requirement and removed outdated output, plus minor corrections.

### CODE REVIEW

File: CONTRIBUTING.adoc
Comment (dduportal) [inline CONTRIBUTING.adoc:9]:
```suggestion
To build the project, make sure that you link:golang[https://go.dev/] installed, link:https://www.gnu.org/software/make/[GNU Make] and link:https://goreleaser.com/install/[goreleaser installed], then execute the command `make build`, or use Docker with:
```
- Remove occurences of `just` (even though the litteral meaning is "only", it sometimes have a deameaning sense for beginners.
- Add reference to GNU make
-  Add link to golang website

File: CONTRIBUTING.adoc
Comment (dduportal) [inline CONTRIBUTING.adoc:17]:
```suggestion
They are multiple ways to contribute which don't necessarily involve coding, like providing feedback, improving documentation and processes.
```

**Review by dduportal — COMMENTED** [review]
Thanks for this improvement! I've let minor comments, and i'll let @olblak approve (and merge) if it is ok

**Review by olblak — APPROVED** [review]
Thank you for this improvement.

**Pull request merged**
